src/api/map/routes/map.ts

```diff
@@ -39,6 +40,27 @@ const list = procedure
     }));
 });
 
+const createLandmarkAssessment = procedure
+  .use(authUser())
+  .input(z.object({
+    userId: z.string(),
+    landmark: z.string(),
+    score: z.number(),
+    markdown: z.string().nullable(),
+  }))
+  .mutation(async ({ input }) => {
+    return await sequelize.transaction(async transaction => {
+      const { userId, landmark, score, markdown } = input;
+      
+      await db.LandmarkAssessment.create({
+        userId,
+        landmark,
+        score,
+        markdown,
+      }, { transaction });
+    });
+  });
+
 const listLandmarkAssessment = procedure
```

Comment:
can you change it to `listLandmarkAssessments` either in this PR or a separate one?